tajo-rpc/tajo-rpc-protobuf/src/main/java/org/apache/tajo/rpc/RpcClientManager.java

```diff
@@ -199,7 +199,6 @@ public static void close() {
    * After it is shutdown it is not possible to reuse it again.
    */
   public static void shutdown() {
-    close();
```

Comment:
This is not directly related to this ticket. It is to solve the hang problem after executing any unit test.